File: c#/City.cs
public class City
{//City class
 //By: Matt Schnider
 
    //Private members
    private string _cityName, _state;
    private double _latitude, _longitude;
    private int _timeZone, _sunSet;
    
    //Constructor
    public City(string cityName, string state, double latitude, double longitude,
                    int timeZone, int sunSet)
    {
        _cityName = cityName;
        _state = state;
        _latitude = latitude;
        _longitude = longitude;
        _timeZone = timeZone;
        _sunSet = sunSet;
    }
    //Public getters 
    public string CityName
    {  
         get{return _cityName;} 
    }
    public string State
    {
        get{return _state;}
    }
    public double Latitude
    {
        get{return _latitude;}
    }
    public double Longitude
    {
        get{return _longitude;}
    }
    public int TimeZone
    {
        get{return _timeZone;}
    }
    public int SunSet
    {
        get{return _sunSet;}
    }
}

File: python/City.py
#Class for City object
#By: Matt Schnider
#
#By nature of python, the member vars are accessible without getter functions. This class, as is 
#allows getting and setting using the .member notation.
class City:
    def __init__(self,cityName,state,latitude,longitude,timeZone, sunSet, passengersWaiting):
        self.CityName = cityName
        self.State = state
        self.Latitude = latitude
        self.Longitude = longitude
        self.TimeZone = timeZone
        self.SunSet = sunSet
        self.PassengersWaiting = passengersWaiting

